src/ManagedNodeAddress.js

```diff
@@ -16,25 +16,22 @@ export default class ManagedNodeAddress {
      */
     constructor(props = {}) {
         if (props.address != null) {
-            const hostAndPortResult = HOST_AND_PORT.exec(props.address);
+            let hostAndPortResult = HOST_AND_PORT.exec(props.address);
```

Comment:
Should have been `const` and a different variable instead to avoid reassigning.
This failed type checker.